errors/arsenalErrors.json

```diff
@@ -463,6 +463,10 @@
      "code": 400,
      "description": "The request was rejected because an invalid or out-of-range value was supplied for an input parameter."
   },
+  "MalformedPolicy": {
+    "code": 400,
+    "description": "Policies must be valid JSON and the first byte must be '{'"
```

Comment:
This description seems to be more like a customized description for that particular JSON parsing error. Is there a more generic one?